src/widgets/ImageRoll.h

```diff
@@ -69,32 +69,4 @@ class AUDACITY_DLL_API ImageRoll
    wxSize       mMaxSize;
 };
 
-// A very simple class that just display an ImageRoll that doesn't
```

Comment:
I think both ImageRoll files could be removed as there are no longer any references to them.

Reply:
I disagree.  Understand whether there are good ideas there worth revisiting before you say that.

And I have never myself studied these files closely enough to answer that.

Reply:
ImageRoll.h is used by AButton.  ExpandingToolBar.* are the unused files.  I suggest don't remove them, but maybe don't modify them except minimally to keep them compiling?